tasks/config.yml

```diff
@@ -357,3 +357,12 @@
     replace: 'PORT=3000 /usr/share/$NODE_VERSION/bin/node main.js'
   when: not bbb_html5_node_options is defined
   notify: restart bigbluebutton
+
+- name: Set recording formats
+  replace:
+    path: /usr/local/bigbluebutton/core/scripts/presentation.yml
+    regexp: '^video_formats:\s*\n\s*#?\s*-\s*webm\n\s*#*\s*-\s*mp4$'
+    replace: |-
+      video_formats:
+        {{ bbb_recordings_webm | ternary('', '#') }}- webm
+        {{ bbb_recordings_mp4 | ternary('', '#') }}- mp4
```

Comment:
IMO Using RegEx for this feels wrong.

This could be done like the Meteor setting:

https://github.com/n0emis/ansible-role-bigbluebutton/blob/56ff2f47f1568c579a963b21517c04eefb7b3ec4/tasks/config.yml#L177-L194

This would make future changes in the `/usr/local/bigbluebutton/core/scripts/presentation.yml` easier.